python/nano/src/bigdl/nano/k8s/bigdl_submit.py

```diff
@@ -0,0 +1,276 @@
+                                   volume_mounts=volumn_mounts)
+
+    volumes = [_deserialize_volume_object(json_str, api_cliet) for json_str in volume_strs]
+
+    pod_spec = client.V1PodSpec(containers=[container],
+                                restart_policy="Never",
+                                volumes=volumes)
+    pod_body = client.V1Pod(api_version='v1',
+                            metadata=metadata,
+                            kind='Pod',
+                            spec=pod_spec)
+    return pod_body
+
+
+def _create_master_service(v1_api: client.CoreApi,
+                           namespace: str,
+                           master_pod_name: str,
+                           master_pod_labels: str,
+                           master_port: str):
+    service_name = f'{master_pod_name}-service'
+    metadata = client.V1ObjectMeta(name=service_name)
+    port = client.V1ServicePort(protocol="TCP",
+                                port=int(master_port),
+                                target_port=int(master_port))
+
+    service_spec = client.V1ServiceSpec(selector=master_pod_labels,
+                                        ports=[port])
+
+    service = client.V1Service(api_version="v1",
+                               kind="Service",
+                               metadata=metadata,
+                               spec=service_spec)
+
+    service = v1_api.create_namespaced_service(namespace=namespace, body=service)
+
+    print(f"Created Master Service: {service_name}")
+    return service_name, master_port
+
+
+def _create_master_pod(v1_api: client.CoreApi,
+                       namespace: str,
+                       pod_name: str,
+                       pod_labels: Dict[str, str],
+                       create_pod_fn: Callable):
+    pod_body = create_pod_fn(rank="0", pod_name=pod_name, pod_labels=pod_labels)
+    pod = v1_api.create_namespaced_pod(namespace=namespace, body=pod_body)
+    print(f"Created Master Pod: {pod_name}")
+
+
+def _create_worker_pods(v1_api: client.CoreApi,
+                        namespace: str,
+                        world_size: int,
+                        app_id: str,
+                        create_pod_fn: Callable):
+
+    for i in range(world_size - 1):
+        pod_name = f'bigdl-{app_id}-worker-{i + 1}'
+        pod_labels = {"bigdl-app": app_id,
+                      "bigdl-app-type": "worker"}
+        pod_body = create_pod_fn(rank=str(i + 1), pod_name=pod_name, pod_labels=pod_labels)
+        pod = v1_api.create_namespaced_pod(namespace=namespace, body=pod_body)
+        print(f"Created Rank {i + 1} Pod: {pod_name}")
+
+
+def main():
+    """Entry point of bigdl-submit command line tool."""
+    args = _parse_args()
+
+    command = ["python", args.main_script] + args.main_script_args
+
+    app_id = str(uuid4())[:7]
+
+    config.load_config()
+
+    v1 = client.CoreV1Api()
+
+    master_pod_name = f'bigdl-{app_id}-master'
```

Comment:
change to driver

Reply:
Ok, but my concern is that PyTorch will use the environmental "MASTER_ADDR" and "MASTER_PORT" and if we change the name here, will it cause confusion?